packages/react-core/src/components/Progress/examples/ProgressTitleOutsideOfProgressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Progress,
+  ProgressMeasureLocation
+} from '@patternfly/react-core';
+
+export const ProgressTitleOutsideOfProgressBar: React.FunctionComponent = () => (
+  <DescriptionList>
+    <DescriptionListGroup>
+      <DescriptionListTerm id="progress-label">Title outside of progress bar</DescriptionListTerm>
+      <DescriptionListDescription>
+        <Progress value={33} measureLocation={ProgressMeasureLocation.outside} aria-labelledby="progress-label" />
```

Comment:
```suggestion
        <Progress value={33} measureLocation={ProgressMeasureLocation.outside} aria-labelledby="title-outside-progress-example-label" />
```

Reply:
Thanks for the feedback! All the required changes have been made. Let me know if there is anything more that needs to be done!